### PR TITLE
Recent work and about me sections

### PR DESCRIPTION
**Pull request by Rifat8080**
- added new 2 sections in mobile versions
- added different projects photos
- added logos of framewroks, skills and languages
- update css with display grid
- added html sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,5 +64,155 @@ <h1>
         </ul>
       </div>
     </section>
+
+    <!-- My recent Works Section -->
+
+    <section>
+      <div class="jobs" id="jobs-id">
```
Comment (PinkMoon25) [inline index.html:71]: - [ ] Excellent work so far, you have done really well :clap: , I just have some suggestions to improve the design, so I think there is supposed to be a gap or spacing between the headline and works section currently it is missing that, please see :point_down: 
image
 figma design :point_down: 
image

I think a margin-top of 80px should fix that :+1:

File: style.css [hunk 2]
```diff
@@ -106,3 +106,220 @@ button {
...
+  justify-content: center;
+}
+
+.box-job > .box-tags > li {
+  background: #ebf0ee;
+  font-size: 12px;
+  padding: 0.5em;
+  margin: 0 0.5em;
+  border-radius: 4px;
+}
```
Comment (PinkMoon25) [inline style.css:183]: - [ ] For these tech tags in the project cards, please fix the color which should be `#3A4A42` , also fix the color of the project card's title i.e. 'Multi-Post_stories' 
image
 figma :point_down: 
image

File: index.html [hunk 3]
```diff
@@ -64,5 +64,155 @@ <h1>
...
+            <img src="Imgs/Recent works/6.jpg" alt="" />
+          </div>
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul class="box-tags">
+            <li>React</li>
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="btn-showup">See Project</button>
```
Comment (PinkMoon25) [inline index.html:151]: - [ ] Please implement the button interactions for all your buttons like :point_down: 
from figma 
image

**Review by PinkMoon25 — CHANGES_REQUESTED** [review]
Hi @Rifat8080 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
Linter checks :heavy_check_mark: 
github flow :heavy_check_mark: 
Flex and Grid :heavy_check_mark: 
Semantics :heavy_check_mark: 
follows figma template :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @Rifat8080, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**